Neos.Neos/Documentation/References/InspectorViewsReference/index.rst

```diff
@@ -0,0 +1,396 @@
+  :alt: A screenshot showcasing the TimeSeriesView. It shows a line graph illustrating fictional page visitor numbers accumulated over time. On its y-axis the graph scales from 400 visitors at the bottom to 1000 visitors at the top. In the top right corner it reads "last month" to signify that the displayed data shows the figures over the last month.
+
+The ``TimeSeriesView`` displays data accumulated over time from a data source as a line chart.
+
+A data source is a PHP class implementing ``Neos\Neos\Service\DataSource\DataSourceInterface`` (which is best achieved by extending ``Neos\Neos\Service\DataSource\AbstractDataSource``). The example displayed above uses the following data source implementation:
+
+.. code-block:: php
+
+  <?php
+
+  /*
+   * This script belongs to the package "Vendor.Site".
+   */
+
+  declare(strict_types=1);
+
+  namespace Vendor\Site\Application\Neos\DataSource;
+
+  use Neos\Neos\Service\DataSource\AbstractDataSource;
+  use Neos\ContentRepository\Domain\Model\NodeInterface;
+
+  final class TimeSeriesViewDataSource extends AbstractDataSource
+  {
+      /**
+      * @var string
+      */
+      static protected $identifier = 'vendor-site-time-series-view';
+
+      /**
+      * Get data
+      *
+      * {@inheritdoc}
+      */
+      public function getData(NodeInterface $node = NULL, array $arguments = [])
+      {
+          return [
+              'data' => [
+                  'rows' => [
+                      [
+                          'date' => new \DateTimeImmutable('2022-09-01'),
+                          'uniqueVisitors' => 503
+                      ],
+                      [
+                          'date' => new \DateTimeImmutable('2022-10-01'),
+                          'uniqueVisitors' => 708
+                      ],
+                      [
+                          'date' => new \DateTimeImmutable('2022-11-01'),
+                          'uniqueVisitors' => 389
+                      ],
+                      [
+                          'date' => new \DateTimeImmutable('2022-12-01'),
+                          'uniqueVisitors' => 1027
+                      ],
+                  ],
+              ]
+          ];
+      }
+  }
+
+.. note:: The data returned from a data source that is used for a ``TimeSeriesView`` need not adhere to a specific shape (as long as it includes a list of data associated with a time and date). The ``viewOptions`` configuration is used to extract data from arbitrary data shapes as you'll see below.
+
+Example ``viewOptions``:
+
+.. code-block:: yaml
+
+  'Vendor.Site:MyCustomNodeType':
+    # ...
+    ui:
+      inspector:
+        views:
+          timeSeriesViewExample:
+            label: 'TimeSeriesView Example'
+            group: examples
+            view: 'Neos.Neos/Inspector/Views/Data/TimeSeriesView'
+            viewOptions:
+              subtitle: 'last month'
+              dataSource: vendor-site-time-series-view
```

Comment:
And here ;)

Reply:
Here it's fine, that is just an identifier…